src/dmd/dsymbolsem.d

```diff
@@ -1157,9 +1158,20 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                 }
                 else
                 {
+                    import dmd.hdrgen : toCBuffer, HdrGenState;
+                    OutBuffer buff;
+                    HdrGenState hgs;
+                    toCBuffer(dsym, &buff, &hgs);
+                    buff.setsize(buff.offset-1);
+
                     // https://issues.dlang.org/show_bug.cgi?id=14166
                     // Don't run CTFE for the temporary variables inside typeof
                     dsym._init = dsym._init.initializerSemantic(sc, dsym.type, sc.intypeof == 1 ? INITnointerpret : INITinterpret);
+                    const init_err = dsym._init.isExpInitializer();
+                    if (init_err && init_err.exp.op == TOK.error)
+                    {
+                        errorSupplemental(dsym.loc, "statical context created here: `%s`", buff.extractString());
```

Comment:
statical => compile time

Reply:
Maybe instead of here, use the actual line number?

Reply:
that would be redundant since errorSupplemental already provides the line number.

Reply:
Hmm imho you should still remove or replace "here" as it's rather a bad word for an error message. Maybe "with"?